nat-lab/tests/test_dns.py

```diff
@@ -16,14 +17,30 @@
 )
 from utils.process import ProcessExecError
 
-DNS_SERVER_ADDRESS = config.LIBTELIO_DNS_IP
+DNS_IPV4_SERVER_ADDRESS = config.LIBTELIO_DNS_IPV4
+DNS_IPV6_SERVER_ADDRESS = config.LIBTELIO_DNS_IPV6
+
+
+class IpVersion(Enum):
+    IPV4 = auto()
+    IPV6 = auto()
 
 
 @pytest.mark.asyncio
-async def test_dns() -> None:
+@pytest.mark.parametrize(
+    "ip_version",
+    [
+        pytest.param(IpVersion.IPV4),
```

Comment:
Why not just parametrize ip itself, instead of ip version, cuz `ip_version` is only used to determine dns_server_address value. For exmaple:

```
@pytest.mark.parametrize("dns_server_address", [DNS_IPV4_SERVER_ADDRESS, DNS_IPV6_SERVER_ADDRESS])
```

Reply:
Look at the description. If you think is better with ip itself, I will change it.

Reply:
I guess it is just a matter of preference :smile:
I think you could also do something like this:
```
@pytest.mark.parametrize(
    "dns_server_address",
    [
        pytest.param(DNS_IPV4_SERVER_ADDRESS, id="IPv4"),
        pytest.param(DNS_IPV6_SERVER_ADDRESS, id="IPv6"),
    ],
)
```
So you wouldn't need introduce additional enum class and you will get your wanted representation

Reply:
Oh, cool. I'll do that.